src/core_plugins/input_control_vis/public/components/editor/field_select.js

```diff
@@ -0,0 +1,73 @@
+import _ from 'lodash';
+import PropTypes from 'prop-types';
+import React, { Component } from 'react';
+import Select from 'react-select';
+
+export class FieldSelect extends Component {
```

Comment:
This component looks like it could be generalized for the uiFramework as well - e.g. FilterableSelect.

@snide - what's your take on components like this. Would you prefer they get moved into the uiFramework ala k6, or should we hold off for now due to incoming k7?